analysis/selectDiphotonEvents.cpp

```diff
@@ -198,7 +212,21 @@ void InitTree(TTree *tr) {
   tr->Branch("costhetastar",        &costhetastar,        "costhetastar/F");
   tr->Branch("cos_photon_pair_helicity0",     &cos_photon_pair_helicity0,     "cos_photon_pair_helicity0/F");
   tr->Branch("cos_photon_pair_helicity1",     &cos_photon_pair_helicity1,     "cos_photon_pair_helicity1/F");
-
+  ////Muon info, Date:29/10/2022
+  tr->Branch("nMu",                  &nMu,                          "nMu/I");
+  tr->Branch("muCharge",             &muCharge,                     "muCharge/I");
+  tr->Branch("muPt",                 &muPt,                         "muPt/F");
+  tr->Branch("muEta",                &muEta,                        "muEta/F");
+  tr->Branch("muPhi",                &muPhi,                        "muPhi/F");
+ // tr->Branch("convertTrack",   &convertTrack,    "converTrack/I"); 
+ // tr->Branch("deltaR1",   &deltaR1,    "deltaR1/F"); 
+ // tr->Branch("deltaR2",   &deltaR2,    "deltaR2/F"); 
```

Comment:
Why is this commented out, shouldn't we store this info?

Reply:
I did this to check converted photons. Later I commented out it.